test/suites/ubuntu-core-edgex-image/test.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash -e
+
+ssh "$USER@localhost" -p 8022 "snap install curl"
```

Comment:
Strangely, I got `snap "curl" is already installed, see 'snap help refresh'` after the first run.

Reply:
Strange! `curl` should not be installed by default. I built the following image from scratch:
```
$ ssh "mengyiw@localhost" -p 8022
$ snap list
Name                  Version         Rev    Tracking       Publisher   Notes
core20                20230207        1828   latest/stable  canonical✓  base
core22                20230210        522    latest/stable  canonical✓  base
edgex-device-virtual  2.3.0           335    latest/stable  canonical✓  -
edgexfoundry          2.3.0+1         4108   latest/stable  canonical✓  -
pc                    22-0.3          x1     -              -           gadget
pc-kernel             5.15.0-66.73.1  1224   22/stable      canonical✓  kernel
snapd                 2.58.2          18357  latest/stable  canonical✓  snapd
$ curl 
-bash: curl: command not found
```